src/cmd1.cc

```diff
@@ -3383,18 +3209,6 @@ void move_player_aux(int dir, int do_pickup, int run, bool_ disarm)
 			if (running) msg_print(NULL);
 		}
 
-		/* Spontaneous Searching */
-		if ((p_ptr->skill_fos >= 50) || (0 == rand_int(50 - p_ptr->skill_fos)))
-		{
-			search();
```

Comment:
Yes, so I think this is what my anti-magic characters without detection were "abusing" to detect hidden doors.